config/converter.go

```diff
@@ -222,3 +223,32 @@ func parseConverterLine(ctx *context, c *Converter, value string) (err error) {
 	}
 	return err
 }
+
+func resolvePkgPath(ctx *context, c *Converter, rawConverter *RawConverter) error {
+	if c.OutputFile == "" {
+		// Nowhere to resolve pkg path from
+		return nil
+	}
+	if c.OutputPackagePath != "" {
+		// User has already set the pkg path
+		return nil
+	}
+
+	absOutputDir := c.OutputFile
+	if !filepath.IsAbs(c.OutputFile) {
+		absOutputDir = filepath.Join(
+			filepath.Dir(rawConverter.FileName),
+			filepath.Dir(c.OutputFile))
```

Comment:
The output:file may include `@cwd` to reference the current working directory. this has to be added here too.